services/graphql-server/src/routes/graphql.js

```diff
@@ -29,12 +30,11 @@ const config = {
   // responses to be multiple megabytes in size!
   tracing: false,
   cacheControl: false,
-  extensions: isProduction ? [() => new ApolloNewrelicExtension()] : [],
-  engine: isProduction ? { apiKey: ENGINE_API_KEY } : false,
+  extensions: NEW_RELIC_ENABLED ? [() => new ApolloNewrelicExtension()] : [],
```

Comment:
This should likely be adjusted to enable per extension - otherwise the entire `extensions` key is based on the `NEW_RELIC_ENABLED` value. Something like...

```js
const config = {
  extensions: [
    ...(NEW_RELIC_ENABLED && [() => new ApolloNewrelicExtension()]),
  ],
};
```